gitops/db.py

```diff
@@ -63,6 +63,14 @@ def copy_db(ctx, source, destination, skip_backup=False, cleanup=True):
 
 
 @task
-def download_backup(ctx, app, index, path=None, datestamp=False):
+def download_backup(ctx, app, index=None, path=None, datestamp=False):
```

Comment:
as we discussed, the side effect of turning this arg into a kwarg will mean users will need to use `-i` now to specify a specific backup number. In this case i think that's fine, cause we'll _always_ want to be using latest image.